client/components/looking-for-team/imports/lft-user-list.jsx

```diff
@@ -2,45 +2,66 @@ import { Meteor } from 'meteor/meteor';
 import { withTracker } from 'meteor/react-meteor-data';
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { Card, Icon } from 'semantic-ui-react';
+import { Card, Icon, Button } from 'semantic-ui-react';
+
+import MessageUserModal from '../../message-user/message-user';
 
 class Users extends Component {
-  render() {
-    if (!this.props.ready) return <Loading/>
-
-    return (
-      <Card.Group>
-        { this._users() }
-      </Card.Group>
-    );
-  }
-
-  _users() {
-    return this.props.users.map((user) => (
-      <Card key={user._id}>
-        <Card.Content>
-          <Card.Header>
-            {user.name}
-          </Card.Header>
-          <Card.Meta>
-            { user.bio }
-          </Card.Meta>
-        </Card.Content>
-        <Card.Content extra>
-          <Icon name='mail'/> <a href={`mailto:${user.getEmail()}`}>{user.getEmail()}</a>
-        </Card.Content>
-      </Card>
-    ));
-  }
+    constructor(props) {
```

Comment:
Warning: nit-pick incoming.

For some reason, this codebase is extremely inconsistent about indentation across different files. This means we can't set a consistent editor setting for the project. As annoying as it is, I like to try to keep the indentation consistent within a file when modifying. This makes the diffs much easier to view. (Note how in this case, the largely unmodified `_users` method is presented as an entirely new block of code, not a modification of the existing method.) emacs (which, given the .gitignore change included here, is what I assume you're using) is certainly not very flexible in this regard, but if you're able to avoid changing indentation when modifying files in the future, it will certainly help out during code review!